healthcareai/supervised_model_trainer.py

```diff
@@ -28,8 +28,10 @@ def __init__(self, dataframe, predicted_column, model_type, impute=True, grain_c
         # Build the pipeline
         # Note: Missing numeric values are imputed in prediction. If we don't impute, then some rows on the prediction
         # data frame will be removed, which results in missing predictions.
-        pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=impute)
-        prediction_pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=True)
+        pipeline = hcai_pipelines.full_pipeline(model_type, predicted_column, grain_column, impute=impute,
```

Comment:
Here verbose should be True to tell the user what is going on during training

Reply:
Training verbose is now set to True.